### PR TITLE
[chore] Move dependabot PRs to Wednesday

### PR DESCRIPTION
**Pull request by dmitryax**
To avoid upgrading dependencies on the release day

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,879 +6,1104 @@ updates:
...
     directory: "/cmd/oteltestbedcol"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/telemetrygen"
     schedule:
       interval: "weekly"
-  - package-ecosystem: "gomod"
-    directory: "/cmd/tracegen"
```
Comment (codeboten) [inline .github/dependabot.yml:38]: Huh... i guess the dependabot check doesn't know to look for components that have been removed? Is that why this hasn't been an issue before?

**Review by codeboten — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (dmitryax) [reply .github/dependabot.yml:38]: I believe it's just `make gendependabot` is not part of the checks CI job

Comment (dmitryax) [reply .github/dependabot.yml:38]: Submitted a PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19405

**Comment by dmitryax** [comment]
@codeboten opened the same for core https://github.com/open-telemetry/opentelemetry-collector/pull/7330

**Pull request merged**